### ISSUE TITLE
keyboard not changing 

### ISSUE THREAD

**Issue by jaithenj** [malaybaku/VMagicMirror#1017]
hello im trying to change the mouse and keyboard and i post it in the accessory folder but nothing show, before it was showing but now after updating its not?!
i did same as guide but nothing work! 
my using V3

**Comment by jaithenj** [comment]
any update?

**Comment by jaithenj** [comment]
any feedback please?

**Comment by malaybaku** [comment]
@jaithenj 
Sorry for late response,

Mouse / Keyboard appearance custom feature refers differenct folder from accessory. It also requires specific file name rule to work.

Please see detail at: https://malaybaku.github.io/VMagicMirror/en/tips/change_textures/

**Comment by jaithenj** [comment]
yes i did that like the guide with name and correct file and as i mentioned it was working perfect but now its not working and even no error show nothing its just dont show the update images im using for mouse oy keyboard
